### ISSUE TITLE
The release build of SDL2-ttf depends on the debug build of SDL2

### ISSUE THREAD

**Issue by ldicker83** [microsoft/vcpkg#9984]
The title basically says it all. The Release builds (both x86 and x64) rely on the debug build of SDL2 (SDL2d.dll not found).

EDIT:
Forgot to mention that I came across #6152 which appears to correct the issue in SDL2-image (I don't see it in that package) but it still appears in SDL2-ttf.

**Comment by DanRStevens** [comment]
I came across #5948, (merged after #6152), which appears to fix the problem for the other SDL2 packages.

Does updating `vcpkg` fix the problem?

**Comment by NancyLi1013** [comment]
Hi @ldicker83
Thanks for posting this issue.
Sorry for the late reply.
Could you please post the fail log about this?
I couldn't reproduce this issue on my local.

@DanRStevens 
Could you please try to update vcpkg to the latest version via` git pull` and build it again?

**Comment by DanRStevens** [comment]
In case I confused anyone, I do most development on Linux, where I use the native package manager, so my question was meant for @ldicker83, whose project I was also working on.

----

I tried pulling a fresh copy of vcpkg in a Windows VM, and everything seemed to build without error. I'm not quite familiar with the original error though, so I'm uncertain if anything was fixed.

I tried backdating to the commit before #5948 was merged and building, hoping I might be able to reproduce a package install error. Unfortunately vcpkg didn't build successfully for that commit, so I wasn't able to reproduce. The build errors mostly involved `experimental::filesystem`, so I suspect the project was setup for an earlier version of Visual Studio back then. I don't plan to investigate further.

At this point, based on lack of any observed error with the current version, I'm going to assume the problem may have been fixed already.
